app/model/commands/UpdateAtomCommand.scala

```diff
@@ -31,7 +31,12 @@ case class UpdateAtomCommand(id: String, atom: MediaAtom)
       AtomNotFound
     }
 
-    val details = atom.contentChangeDetails.copy(revision = existingAtom.get.contentChangeDetails.revision + 1)
+    val changeRecord = ChangeRecord(DateTime.now(), None)
```

Comment:
Completely agree with the abstracting, can do this when we add Users in (I have a list of Todos!)